File: Baekjoon_step/Backjoon4673/Backjoon4673.c
#include <stdio.h>

void selfnum(){
    int a[10000] = {0}; 

    for(int i = 0; i < 10; i++)
        for(int ii = 0; ii < 10; ii++)
            if(i*11+ii*2 < 100)
                a[i*11+ii*2] = 1;

    for(int i = 0; i < 10; i++)
        for(int ii = 0; ii < 10; ii++)
            for(int iii = 0; iii < 10; iii++)
                if(i*101+ii*11+iii*2 < 1000)
                    a[i*101+ii*11+iii*2] = 1;

    for(int i = 0; i < 10; i++)
        for(int ii = 0; ii < 10; ii++)
            for(int iii = 0; iii < 10; iii++)
                for(int iiii = 0; iiii < 10; iiii++)
                    if(i*1001+ii*101+iii*11+iiii*2 < 10000)
                        a[i*1001+ii*101+iii*11+iiii*2] = 1;

    for(int i = 1; i < 10000; i++)
        if(a[i] == 0) 
            printf("%d\n",i); 
}

int main(){
    selfnum();
    return 0;
}

File: Backjoon/Backjoon4673/Backjoon4673.py
#include <stdio.h>

def selfnum():
    a = list()
    for i in range(0, 10000):
        a.append(0) 

    for i in range(0, 10):
        for ii in range(0, 10):
            if(i*11+ii*2 < 100):
                a[i*11+ii*2] = 1

    for i in range(0, 10):
        for ii in range(0, 10):
            for iii in range(0, 10):
                if(i*101+ii*11+iii*2 < 1000):
                    a[i*101+ii*11+iii*2] = 1

    for i in range(0, 10):
        for ii in range(0, 10):
            for iii in range(0, 10):
                for iiii in range(0, 10):
                    if(i*1001+ii*101+iii*11+iiii*2 < 10000):
                        a[i*1001+ii*101+iii*11+iiii*2] = 1

    for i in range(1, 10000):
        if(a[i] == 0): 
            print(i)

selfnum()